I'm seeing "can't find git" errors, apparently from one of the sbt commands.

```
+ docker run --rm -v /Users/thibault/src/firecloud-orchestration:/working -w /working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage /working/src/docker/install.sh /working
[info] Loading project definition from /working/project
Exception in thread "Thread-1" java.io.IOException: Cannot run program "git": error=2, No such file or directory
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
        at sbt.SimpleProcessBuilder.run(ProcessImpl.scala:349)
        at sbt.SequentialProcess.runAndExitValue(ProcessImpl.scala:240)
        at sbt.CompoundProcess$$anonfun$1.apply$mcV$sp(ProcessImpl.scala:201)
        at sbt.Spawn$$anon$3.run(ProcessImpl.scala:17)
Caused by: java.io.IOException: error=2, No such file or directory
        at java.lang.UNIXProcess.forkAndExec(Native Method)
        at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
        at java.lang.ProcessImpl.start(ProcessImpl.java:134)
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
        ... 4 more
[info] Set current project to FireCloud-Orchestration (in build file:/working/)
```

--- a/project/Version.scala
+++ b/project/Version.scala
@@ -1,41 +1,26 @@
 import sbt.Keys._
 import sbt._
 
+
 object Version {
-  val versionRoot = "0.1"
+  val baseModelVersion = "0.1"
 
   def getVersionString = {
-    // Get the revision, or -1 (later will be bumped to zero)
-    val versionRevision = {
-      try {
-        ("git rev-list --count HEAD" #|| "echo -1").!!.trim.toInt
-      } catch {
-        case e: Exception =>
-          0
-      }
-    }
+    def getLastModelCommitFromGit = { s"""git rev-parse --short HEAD""" !! }
 
-    // Set the suffix to None...
-    val versionSuffix = {
-      try {
-        // ...except when there are no modifications...
-        if ("git diff --quiet HEAD".! == 0) {
-          // ...then set the suffix to the revision "dash" git hash
-          Option(versionRevision + "-" + "git rev-parse --short HEAD".!!.trim)
-        } else {
-          None
-        }
-      } catch {
-        case e: Exception =>
-          None
-      }
-    }
+    // either specify git model hash as an env var or derive it
+    // if building from the broadinstitute/scala-baseimage docker image use env var
+    // (scala-baseimage doesn't have git in it)
+    val lastModelCommit = sys.env.getOrElse("GIT_MODEL_HASH", getLastModelCommitFromGit ).trim()
+    val version = baseModelVersion + "-" + lastModelCommit
 
-    versionRoot + "-" + versionSuffix.getOrElse((versionRevision + 1) + "-SNAPSHOT")
+    // The project isSnapshot string passed in via command line settings, if desired.
+    val isSnapshot = sys.props.getOrElse("project.isSnapshot", "true").toBoolean
 
+    // For now, obfuscate SNAPSHOTs from sbt's developers: https://github.com/sbt/sbt/issues/2687#issuecomment-236586241
+    if (isSnapshot) s"$version-SNAPSHOT" else version
   }
 
   val rootVersionSettings: Seq[Setting[_]] =
     Seq(version := getVersionString)
-
 }

--- a/script/build.sh
+++ b/script/build.sh
@@ -7,20 +7,26 @@
 #   jar : build orch jar
 #   -d | --docker : provide arg either "build" or "push", to build and push docker image
 # Jenkins build job should run with all options, for example,
-#   ./docker/build.sh jar -d push
+#   ./script/build.sh jar -d push
 
 set -ex
 PROJECT=firecloud-orchestration
 
 function make_jar()
 {
-    docker run --rm -v $PWD:/working -w /working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 broadinstitute/scala-baseimage /working/src/docker/install.sh /working
+    # Set for versioning the jar
+    GIT_MODEL_HASH=$(git rev-parse --short ${GIT_BRANCH})
+
+    docker run --rm -e GIT_MODEL_HASH=${GIT_MODEL_HASH} \
+        -v $PWD:/working -w /working -v jar-cache:/root/.ivy -v jar-cache:/root/.ivy2 \
+        broadinstitute/scala-baseimage /working/src/docker/install.sh /working
 }
 
 function docker_cmd()
 {
-    if [ $DOCKER_CMD="build" ] || [ $DOCKER_CMD="push" ]; then
+    if [ $DOCKER_CMD = "build" ] || [ $DOCKER_CMD = "push" ]; then
         echo "building docker image..."
+
         GIT_SHA=$(git rev-parse ${GIT_BRANCH})
         echo GIT_SHA=$GIT_SHA > env.properties  # for jenkins jobs
         docker build -t $REPO:${GIT_SHA:0:12} .

